tests/app/utilities/test_schema.py

```diff
@@ -41,20 +41,24 @@ def test_get_allowed_languages(schema_name, launch_language, expected):
 def test_get_schema_path_map():
     schema_path_map = get_schema_path_map(include_test_schemas=True)
 
-    assert all(
-        language_code in schema_path_map.keys() for language_code in ["en", "cy"]
-    )
-    assert all(os.path.exists(path) for path in schema_path_map["en"].values())
-    assert all(
-        os.path.basename(path).replace(".json", "") == schema_name
-        for schema_name, path in schema_path_map["en"].items()
-    )
+    assert "test" in schema_path_map
```

Comment:
There is a lot going on in this test. I am thinking it asserts test schemas are there that "en" and "cy" are created for each survey type and that the path is correct for each? Because there is so much going on and its not immediately obvious I think a comment(s) would help

At the moment it's also including social and business, should we be including them? (not sure) Also how it's implemented we are generating ga for each, should that be included or should be retire ga for good, as at the moment there is a gap.